src/context/ReputationChartContext/ReputationChartContext.ts

```diff
@@ -0,0 +1,20 @@
+import { createContext, useContext } from 'react';
+
+export const ReputationChartContext = createContext<
+  | {
+      setActiveLegendItem: (id: string) => void;
+      resetActiveLegendItem: () => void;
```

Comment:
technically we don't need this one anymore, but it may be useful to keep it down the line

Reply:
Yeah, I was thinking about removing it, but thought we might make use of resetting the active legend item at some point 😅